src/common/layer.js

```diff
@@ -17,9 +17,9 @@ const $ = jQuery;
  * an existing group element or, with three parameters, will create a new layer group element.
  *
  * @example
- * const l1 = new Layer('name', group); // Use the existing group for this layer.
- * const l2 = new Layer('name', group, svgElem); // Create a new group and add it to the DOM after group.
- * const l3 = new Layer('name', null, svgElem); // Create a new group and add it to the DOM as the last layer.
+ * l1 = new Layer('name', group); // Use the existing group for this layer.
+ * l2 = new Layer('name', group, svgElem); // Create a new group and add it to the DOM after group.
+ * l3 = new Layer('name', null, svgElem); // Create a new group and add it to the DOM as the last layer.
```

Comment:
Why is this passing linting? I would expect the rule against undeclared variables should still apply. I'd prefer to keep it applying also.

Reply:
Good point. The previous code was actually not passing eslint which I found strange. However, as it was an example, I took the shortcut to remove the declaration as it was solving the issue.

Reply:
fixed